src/output/stream_writer.rs

```diff
@@ -0,0 +1,63 @@
+use std::os::raw::{c_char, c_int};
+use std::io::{Write, Seek, SeekFrom};
+use std::slice;
+
+/// A pointer to this is passed to the OpenEXR C++ API for writing
+/// to the IO source it represents.  It hides T from the C++ API
+/// and also keeps track of the cursor position, which Rust's Seek
+/// trait doesn't expose.
+///
+/// Note: the reason we can't just pass the pointer to T directly
+/// is because it could be a fat pointer to a trait object.
```

Comment:
Oh, that's weird.  Apparently I left a dead file in.  That's from an earlier iteration, and is dead code.  I realized the same thing, and got rid of that struct entirely.  I'll remove the file.  Sorry!